### DISCUSSION TITLE
Idea for new Patch/Map Change (Dungeon/Cavern Map)

### DISCUSSION THREAD

**Discussion by ryotgg** [ValveSoftware/Dota2-Gameplay#6397] [Ideas]
Hi everyone!

![dota-2-cave](https://user-images.githubusercontent.com/85581823/210803657-19be22f2-173e-4625-8893-c4a7c74ebc0d.png)

I woke up 3a.m today with this crazy idea for a new patch and map change.
I always think about **how to make Roshan balanced between dire and radiant.** 

Than this idea came out. It would be fun if there was another map inside the normal map (like the pirate map of HOTS). 
Roshan lives in a small pit... **what if we put him in a real cave?** 

With more space to play, Roshan could be a real (raid) boss, like the ones in HON, with more skills and mechanics.
He even could gain difficult levels for each death.

The cave or dungeon have 2 or 4 entrances. 
It's a separeted map and who is inside can't se the above map. 

Inside the cave is some Cave Creeps (like Jungle Creeps), that increases the complexity of farm routes and adds more possibilities. 
If the enemy invades your side of map, your hc can farm the cave and go direct to the enemy jungle, i.e.

The counter part is that you can get ganked easily because you can't see where the enemies are coming, and will depend on team communication.

The outposts could be used to give vision in the cave entrances (inside or outside idk). 

(optional) Another mechanic is the stalagmites and stones inside the cave, that can be used for fog/jukes and broken with a new item: Pickaxe. 

Issues: 
-Getting inside of cave for ganks is broken, because it would be like a smoke.  The enemy will pop in your jungle and you can't prevent that. How to solve that? I think the enemy triangle entrance can be blocked until you get some objective (like an outpost) outside, that is avaliable when you destroy the first T2 like today.
-How about wards inside? You can only see the ward vision when you are inside the cave? Or there will be 2 maps in the UI? 

That's it. 
Everyone who listened to the idea liked it and I wanted to give it a try here.

**Comment by commentator24** [comment]
griefing players would enter cave and never come out after digging with pickaxe. 🙄 🙄 🙄 ..🤣 

Gives me an idea for a cavern crawl though. Players waiting to respawn could have something to do e.g. only during 10-90 seconds death timer can access cavern in your Ethereal form to dig your way and find soul shards. you also receive 1 soul shard (like leveling talent) per death to modify your effigy. When effigy in base is destroyed, it will summon an ancestor of your hero like mini roshan to help you defend throne.

possible scenarios:
mega creeps = all effigies auto break and summon all 5 ancestor heroes to help defend throne. ancestors are not that powerful, just as strong as a mega creep, but these units are immortals, cannot die like greevils in diretide and may summon more minions. creeps ignore these units and go straight for throne.
Winning team always have weaker effigy if they dont die / teamfight often.
Winning team can ignore destroying effigies / prevent summon if they go straight for throne.
Cavern has designated 10 effigies locations. Can weaken enemy effigy or strengthen yours if you wish.
Shovel in cavern to get points / nothing. chance based.
cavern portals are only accessible for 20 seconds every 5 min like underlord ult. players can use this time to quickly plan where to go.
Or portal entry / exit points are T1, T2, T3 after each tower destroyed. Cant stay too long in cavern, after 40 seconds will be kicked out to where you entered if you cant find exit.
